src/registry.json

```
@@ -3597,6 +3597,37 @@
         }
       }
     },
+    "npm:@web3-name-sdk/snap": {
+      "id": "npm:@web3-name-sdk/snap",
+      "metadata": {
+        "name": "SPACE ID Web3 Domain",
+        "author": {
+          "name": "SPACE ID",
+          "website": "https://space.id/"
+        },
+        "website": "https://snap.space.id/",
+        "summary": "The SPACE ID Web3 Domain Snap enables the resolution of web3 domains in MetaMask, such that you can type in domain names when you transfer crypto.",
+        "description": "After installing the SPACE ID Web3 Domain Snap, when transferring crypto, you can type in any web3 domains names you registered on SPACE ID directly instead of copying & pasting complex wallet addresses.",
+        "audits": [
+          {
+            "auditor": "SlowMist",
+            "report": "https://github.com/slowmist/Knowledge-Base/blob/master/open-report-V2/blockchain-application/SlowMist%20Audit%20Report%20-%20web3-name-snap_en-us.pdf"
+          }
+        ],
+        "category": "name resolution",
+        "support": {
+          "contact": "https://discord.com/channels/953120695431331901/978354253838049371",
```

This doesn't work.

---

Good catch!
```suggestion
          "contact": "https://discord.gg/spaceid",
```
Wondering if we can just do this, this is linked in the docs.